lib/lte_link_control/modules/ncellmeas.c

```diff
@@ -783,6 +793,24 @@ int ncellmeas_start(struct lte_lc_ncellmeas_params *params)
 	return err;
 }
 
+static void ncellmeas_cancel_timeout_work_fn(struct k_work *work)
+{
+	struct lte_lc_evt evt = {0};
+
+	if (k_sem_count_get(&ncellmeas_idle_sem) == 0) {
+		LOG_DBG("No %%NCELLMEAS notification received after AT%%NCELLMEASSTOP, "
```

Comment:
Thinking out loud. Would this be something we wanted to see if it occurs so maybe `LOG_INF`?
This should be a rather rare case in real life so it's not going to spam the log.
We would especially want to see this log if the notification happens to come later than 1s.

Reply:
We currently have only two info logs in LTE LC, of which one should rather be a warning, IMHO. While this probably means that there could be more info level logging, I'm not sure this is an example of such a case. This is (or should be) a rare case, but is the overall user is interested in this information? If it's mainly intended for the developers of the library, the current log level might be the right one.